src/sagemaker/jumpstart/factory/model.py

```diff
@@ -730,16 +780,16 @@ def get_init_kwargs(
     # we use the model artifact from the training job output
     if not model_from_estimator:
         model_init_kwargs = _add_model_data_to_kwargs(kwargs=model_init_kwargs)
-
     model_init_kwargs = _add_source_dir_to_kwargs(kwargs=model_init_kwargs)
     model_init_kwargs = _add_entry_point_to_kwargs(kwargs=model_init_kwargs)
     model_init_kwargs = _add_env_to_kwargs(kwargs=model_init_kwargs)
     model_init_kwargs = _add_predictor_cls_to_kwargs(kwargs=model_init_kwargs)
     model_init_kwargs = _add_extra_model_kwargs(kwargs=model_init_kwargs)
     model_init_kwargs = _add_role_to_kwargs(kwargs=model_init_kwargs)
-
     model_init_kwargs = _add_model_package_arn_to_kwargs(kwargs=model_init_kwargs)
 
     model_init_kwargs = _add_resources_to_kwargs(kwargs=model_init_kwargs)
 
+    _log_model_type(kwargs=model_init_kwargs)
```

Comment:
sanity-check: why do we need this info log? Just want to make sure it isn't a left-over debug statement.

Reply:
I discussed with Evan that we want to have some logging indicating a proprietary model is used. Now we log another warning that is displayed to the customer, so I think it's not necessary to keep this one. Removed.